EmployeeLeaveTracking/EmailService/EmailHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Net.Mail;
+using System.Net;
+
+namespace EmailService
+{
+    public class EmailHelper
+    {
+
+        public void SendEmail(string emailToAddress,string subject, string body)
+        {
+            using MailMessage mail = new();
+            string smtpAddress = "smtp.gmail.com";
```

Comment:
Move all these configurations to config files

Reply:
Added Configurable Values

Reply:
Made the email helper values configurable